inc/azure_uamqp_c/amqpvalue.h

```diff
@@ -92,6 +92,8 @@ extern "C" {
 
     MOCKABLE_FUNCTION(, int, amqpvalue_encode, AMQP_VALUE, value, AMQPVALUE_ENCODER_OUTPUT, encoder_output, void*, context);
     MOCKABLE_FUNCTION(, int, amqpvalue_get_encoded_size, AMQP_VALUE, value, size_t*, encoded_size);
+    MOCKABLE_FUNCTION(, int, amqpvalue_encode_array_item, AMQP_VALUE, item, bool, first_element, AMQPVALUE_ENCODER_OUTPUT, encoder_output, void*, context);
```

Comment:
>amqpvalue_encode_array_item [](start = 29, length = 27)

I think these are internal to the amqpvalue module.
A user of the module would always encode an entire array, so they should be removed from the .h and made static.

Reply:
I mean both of them: amqpvalue_encode_array_item and amqpvalue_get_encoded_array_item_size

---
In reply to: [230628469](https://github.com/Azure/azure-uamqp-c/pull/271#discussion_r230628469) [](ancestors = 230628469)